### ISSUE TITLE
Dragging texture to normal slot on material causes editor to crash.

### ISSUE THREAD

**Issue by jayrulez** [BearishSun/BansheeEngine#248]
Furthermore, I am unable to reopen the project successfully after the crash.

[BsTest3.zip](https://github.com/BearishSun/BansheeEngine/files/2493753/BsTest3.zip)

I've attached the project so you can check the behaviour.

**Comment by jayrulez** [comment]
Similar crash: I created a new project, added a quad to the scene, created a material then saved the project and closed.

I then reopened the project and attempted to click on the material.

This causes a crash.

**Comment by BearishSun** [comment]
Thanks. I'm currently focusing on some framework stuff, but will get back to editor bug fixing in a week or two.

**Comment by christianclavet** [comment]
Hi!

Seem to work here. I deleted the "internal" folder, then re-opened the project and here the screen:
![image](https://user-images.githubusercontent.com/13395943/56087457-d28b4480-5e39-11e9-93a8-a3f7d00646e2.png)

NOTE:
One thing that I is not working properly is that materials don't seem to be properly updated. I had to exit the editor and reload the scene to have the material showing properly (there were some changes done recently to the material system, so probably need some more tweaks)
